src/main/java/com/czertainly/api/model/client/approval/ApprovalDto.java

```diff
@@ -0,0 +1,86 @@
+package com.czertainly.api.model.client.approval;
+
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Data
+@NoArgsConstructor
+public class ApprovalDto {
+
+    @Schema(
+            description = "UUID of the Approval",
+            requiredMode = Schema.RequiredMode.REQUIRED
+    )
+    private String approvalUuid;
+
+    @Schema(
+            description = "Version of the Approval profile",
+            requiredMode = Schema.RequiredMode.REQUIRED
+    )
+    private int version;
+
+    @Schema(
+            description = "Creation date of the Approval",
+            requiredMode = Schema.RequiredMode.REQUIRED
+    )
+    private Date createdAt;
+
+    @Schema(
+            description = "Date of resolution of the Approval",
+            requiredMode = Schema.RequiredMode.NOT_REQUIRED
+    )
+    private Date closedAt;
+
+    @Schema(
+            description = "Status of the Approval",
+            requiredMode = Schema.RequiredMode.REQUIRED
+    )
+    private String status;
```

Comment:
Use corresponding enum type and change enum that implements IPlatformEnum interface and is included in PlatformEnum.